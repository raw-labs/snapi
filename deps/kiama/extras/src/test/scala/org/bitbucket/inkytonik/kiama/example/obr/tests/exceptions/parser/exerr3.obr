(* Missing DO in CATCH *)

PROGRAM ExErr3 (i : INTEGER) : INTEGER;

BEGIN

   TRY
      Average := Quantity / Count;         
      Result := Average * Frequency;
   CATCH DivideByZero
      Result := 0;
   END
   
END ExErr3.
