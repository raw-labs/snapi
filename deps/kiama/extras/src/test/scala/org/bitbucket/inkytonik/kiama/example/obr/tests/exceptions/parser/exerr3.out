src/test/scala/org/bitbucket/inkytonik/kiama/example/obr/tests/exceptions/parser/exerr3.obr:11:7:error: 'DO' expected but 'R' found
      Result := 0;
      ^
